templates/form/index.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello FormController!{% endblock %}
+
+{% block body %}
+
+    <div class="bg-light m-4">
+        <table class="table">
+            <tbody>
+            <tr>
+                <th scope="row">Projektas</th>
+                <th scope="row">Studentas</th>
+            </tr>
+            <tr>
+                <td>{{ project|escape }}</td>
```

Comment:
Symfony įprastinis nustatymas yra – viską `escape`inti. Tai papildomai to rašyti nebūtina.

> If you're using Twig templates, then output escaping is on by default.
https://symfony.com/doc/current/templating/escaping.html#output-escaping-in-twig

Plačiau:
 * https://twig.symfony.com/doc/2.x/filters/escape.html|
 * https://twig.symfony.com/doc/2.x/filters/raw.html